packages/blocks/shuffle/src/components/item.styled.ts

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
Wanted to keep the item file shorter and less distracting since it already has lots of logic. I don't see a benefit to keep the styled components at the top of the file, especially when they are long like this. I like the idea of keeping styles in a different file unless they are really short and did not make the `jsx` harder to read & understand, or did not crowd the component file that much.